packages/Babylonian-UI.package/BPExampleMorph.class/instance/deleteAllAnnotationsClicked.st

```diff
@@ -2,12 +2,13 @@ actions
 deleteAllAnnotationsClicked
 
 	"Deletes all annotations for this example."
-	| browser annotations |
-	browser := BPBrowser open.
-	annotations := self example trace probes.
-	annotations addAll: self example trace assertions.
-	annotations do: [:annotationTraceValues | 
-		browser browseReference: annotationTraceValues first containedMethodReference.
-		browser removeAnnotations: {annotationTraceValues first belongingAnnotation}].
-	
-	browser currentWindow delete
\ No newline at end of file
+	| browser annotations |	
+	self userWantsToDeleteAllAnnotations ifTrue: [
```

Comment:
nitpick: or `self userWantsToDeleteAllAnnotations ifTrue:[^ self]` to save a tab everywhere?